src/components/UpcomingEventsList.tsx

```diff
@@ -50,6 +50,12 @@ const UpcomingEventsList: React.FC<IProps> = () => {
     )
   }
 
+  const formatDate = (date: Date): string => {
+    return (
+      date.getFullYear() + "-" + (date.getMonth() + 1) + "-" + date.getDate()
+    )
+  }
+
```

Comment:
Hey @ameeetgaikwad, thanks for the PR and sorry for delays... JavaScript has a library called `Intl` which should be used here. It allows us to pass it the users locale (language/region) to format the date appropriately for their region.

Examples:
<img width="353" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/04d7ca0d-ff21-4f13-8dd6-036814eea8d4">

For this repo, we're right in the middle of migrating from using Gatsby to using Next.js... SO, I wouldn't update this _right now_, but very soon when the [Next.js repo fork](https://github.com/ethereum/ethereum-org-fork) has been merged into this repo (within 1-2 weeks), we can access the users "locale" using the `useRouter` hook:

```tsx
// example use
import { useRouter } from "next/router"
// ...then inside a component function:
  const { locale } = useRouter()
// ...then use this locale with the built-in Intl.DateTimeFormat function to format `date` accordingly
  Intl.DateTimeFormat(locale).format(date)
```

To avoid redundancy, you could makes a helper function inside the component so it has access to `locale`, something like this:

```tsx
  const formatDate = (date: Date): string => new Intl.DateTimeFormat(locale).format(date)
```

Then use `formatDate` below the same way you are currently doing.

Reply:
@wackerow  Sure, I'll wait for the merge. 

Also rather than using `formatDate` can we like this?: 
![image](https://github.com/ethereum/ethereum-org-website/assets/76063569/507ac6c0-f074-4f4c-9cb2-eed741f0ec83)

Reply:
I would suggest using the `Intl` package here (although admittedly results would be similar, we've just been trying to standardize around using the `Intl` package for these types of things)...

Ideally we pull this out as a global helper function since we could use it in multiple places on the site. Again, _after_ the Next.js migration is complete (moving along, should be ready very soon), I would look to add this to `src/lib/utils/date.ts` as a new exported function where it takes in a date and a locale, and returns the formatted date string for that locale. Then we can import that wherever we need it to reduce redundancy.

Reply:
@wackerow sure then I'll add the function in `src/lib/utils/date.ts` . And use that function here.

Reply:
Hey @ameeetgaikwad! Just checking in, were you still interested in making these updates? If not no worries, can assign someone else, just checking